src/kirby/components/list/list-item-color.directive.ts

```diff
@@ -0,0 +1,22 @@
+import { Directive, Input, Renderer2, ElementRef, OnChanges, SimpleChanges } from '@angular/core';
+
+@Directive({
+  selector: '[kirbyListItemColor]',
+})
+export class ListItemColorDirective implements OnChanges {
+  @Input()
+  colorCb: (item: any) => string;
+
+  @Input() item: any;
+
+  constructor(private renderer: Renderer2, private elementRef: ElementRef) {}
+
+  ngOnChanges(changes: SimpleChanges): void {
+    if (!this.colorCb || !this.item) {
+      return;
+    }
+
+    const color = this.colorCb(this.item);
+    this.renderer.addClass(this.elementRef.nativeElement, color);
```

Comment:
Why not just use `@HostBinding` instead of Renderer2?

Reply:
Renderer2 is less code and is more convenient when we want to set an unknown class than hostBinding.